libs/elasticsearch/tests/fake_embeddings.py

```diff
@@ -15,19 +15,13 @@ def embed_documents(self, texts: List[str]) -> List[List[float]]:
         Embeddings encode each text as its index."""
         return [[float(1.0)] * 9 + [float(i)] for i in range(len(texts))]
 
-    async def aembed_documents(self, texts: List[str]) -> List[List[float]]:
-        return self.embed_documents(texts)
-
```

Comment:
being able to test async is useful. Probably keep this?

Reply:
We currently don't test against the async versions in the tests, so I removed it from these test classes as a cleanup.